### PR TITLE
Update ar.json

### PR DESCRIPTION
**Pull request by MuhAssar**
support latest primeng v17.18

### CODE REVIEW

**Comment by melloware** [comment]
Going to reject this. All the labels you removed are in use in PrimeFaces JsF and PrimeReact.  PrimeNG needs to be update to use all these labels your removed

Please only add the missing primeNG labels and make sure they are in alphabetical order

**Review by melloware — CHANGES_REQUESTED** [review]
See comments

**Comment by MuhAssar** [comment]
I revert the deletion of the labels that were not used in PrimeNG, and sorted the new additions in alphabetical order

File: ar.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
   "ar": {
+    "after": "بعد",
+    "before": "قبل",
+    "is": "يكون",
+    "isNot": "ليس",
```
Comment (melloware) [inline ar.json:6]: These are wrong PrimeNG should be using the following which are already defined in the file...
```
    "dateAfter": "التاريخ بعد",
    "dateBefore": "التاريخ قبل",
    "dateFormat": "شهر/يوم/سنة",
    "dateIs": "التاريخ هو",
    "dateIsNot": "التاريخ ليس",
```

See PrimeNG bug: https://github.com/primefaces/primeng/issues/14446

File: ar.json [hunk 2]
```diff
@@ -163,14 +170,17 @@
       "pageLabel": "صفحة {page}",
       "passwordHide": "اخفاء كلمة المرور",
       "passwordShow": "عرض كلمة المرور",
+      "prevPageLabel": "الصفحة السابقة",
```
Comment (melloware) [inline ar.json:173]: This is incorrect there is already `previousPageLabel` please report to PrimeNG. See: https://github.com/primefaces/primelocale/pull/170

**Review by melloware — CHANGES_REQUESTED** [review]

**Comment by MuhAssar** [comment]
looks like PrimeNG is not synced with the rest of PrimeTek in regards to this file schema

**Comment by melloware** [comment]
@MuhAssar yes I agree.

**Pull request closed without merging**